torch/csrc/utils/pycfunction_helpers.cpp

```diff
@@ -0,0 +1,5 @@
+#include "pycfunction_helpers.h"
+
+PyCFunction convertPyCFunctionWithKeywords(PyCFunctionWithKeywords func) {
```

Comment:
Ho I actually meant the signature from the python standard that has the right number of arguments: https://docs.python.org/3/c-api/structures.html#c.PyCFunctionWithKeywords

Reply:
Right although we still need to cast to PyCFunction instead PyMethodDef needs a PyCFunction type right?

Reply:
Ho I though having the `PyCFunctionWithKeywords` type was enough. but you do need this extra cast indeed.